app/views/features/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <h2>Features</h2>
 
-<p>Features allow admins to edit/update text and images displayed around the site, for example banners, sidebar content, and alerts or notices.</p>
+<p style="margin-left:10px;margin-top:0.5rem;";>Features allow admins to edit/update text and images displayed around the site, for example banners, sidebar content, and alerts or notices.</p>
```

Comment:
This CSS is not fluid, rather fixed which needs to be avoided. 1px is not the same on normal and MacOS screens.
Have you tried `vertical-align: middle;`?